examples/event-interceptors/README.md

```diff
@@ -0,0 +1,25 @@
+## Sample Event Interceptors
+
+This folder contains some sample event interceptors:
+
+* [Github Validation](#github-validation)
+
+
+### Github Validation
+
+Validates Github webhook payloads using a supplied webhook secret token. Assumes 
```

Comment:
Nit: Github -> GitHub

Reply:
Added a comment to change this. The task does allow the parameter to be changed.

Reply:
Is it worth it to have both a task as well as s deployment? A user could choose to use either. Maybe having both and that too in two different places is confusing? What do you think @a-roberts ?

Reply:
I like the idea of a Tekton Task yeah, users get to see the logs e.g. through the dashboard or kubectl trivially, they have a "record" (in the form of a TaskRun CRD) of the interceptor being created (which might be useful on a prod system where serviceability is an important factor) and it serves as another example of how a good Tekton Task definition should look, so I'm all for having both

Reply:
An example TaskRun would be super useful here too btw